### ISSUE TITLE
cross compile failed on m68k

### ISSUE THREAD

**Issue by shipitsin-ilia** [haproxy/haproxy#2052]
### Tool Name and Version

gcc

### Code Report

```plain
just in case it was missed

https://github.com/haproxy/haproxy/actions/runs/4227971869/jobs/7342934051

  CC      src/quic_conn.o
In file included from include/haproxy/quic_tls.h:30,
                 from src/quic_conn.c:60:
src/quic_conn.c: In function ‘quic_rx_pkts_del’:
include/haproxy/trace.h:134:65: error: format ‘%zu’ expects argument of type ‘size_t’, but argument 6 has type ‘uint64_t’ {aka ‘long long unsigned int’} [-Werror=format=]
  134 |                         _msg_len = snprintf(_msg, sizeof(_msg), (fmt), ##args); \
      |                                                                 ^~~~~
include/haproxy/trace.h:127:9: note: in expansion of macro ‘TRACE_PRINTF_LOC’
  127 |         TRACE_PRINTF_LOC(level, mask, ist(TRC_LOC), __FUNCTION__, a1, a2, a3, a4, fmt, ##args)
      |         ^~~~~~~~~~~~~~~~
src/quic_conn.c:6582:17: note: in expansion of macro ‘TRACE_PRINTF’
 6582 |                 TRACE_PRINTF(TRACE_LEVEL_DEVELOPER, QUIC_EV_CONN_LPKT, qc, 0, 0, 0,
      |                 ^~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
cc1: all warnings being treated as errors
make: *** [Makefile:988: src/quic_conn.o] Error 1
Error: Process completed with exit code 2.
```

### Output of `haproxy -vv`

```plain
no
```

**Comment by haproxyFred** [comment]
I have just pushed a fix here for this issue:

https://github.com/haproxytech/quic-dev/commit/ef67174b8a7ae5dc9c87b42f4e02d278a3121bcc

**Comment by Darlelet** [comment]
The fix was merged in master: b7a13be6cd20b3840ae516fc423bae27576314da
